src/components/Auth/Forms/AuthForm/UserForm.jsx

```diff
@@ -15,29 +15,37 @@ const UserForm = ({
   name,
   extras,
 }) => {
-  const token = localStorage.getItem('token')
   const [errMsg, setErrMsg] = useState('')
-  const [login, { isLoading, error, isError }] = useLogInMutation()
+  const [authUser, { isLoading }] = useAuthUserMutation()
```

Comment:
this name isn't descriptive enough... I believe its a `login` function so why change the name to `authUser`?